roles/test-beat/tasks/common/assert.yml

```diff
@@ -47,3 +47,15 @@
       - "registry_stat.stat.exists"
       - "registry_stat.stat.size > 0"
   when: registry_file != ''
+
+- name: 'Get {{ beat_name }} metrics from logs'
+  shell: 'grep "Total non-zero metrics" {{ beat_log_file }} | tail -1'
+  register: log_metrics
+  when: ansible_system != "Win32NT"
```

Comment:
You can omit these checks because `common` really means POSIX. So this file never gets used on Windows.